src/gdext/surface/userenums.nim

```diff
@@ -13,9 +13,16 @@ proc registerEnumFields*(className, enumName: StringName; fields: varargs[tuple[
 template registerEnumFields*[T: SomeUserClass](Class: typedesc[T]; enumName: StringName; fields: varargs[tuple[name: StringName; value: Int]]; isBitField: bool) =
   registerEnumFields(className Class, enumName, fields, isBitField)
 
-proc registerEnumInternal(Class, Enum: NimNode; isBitField: bool): NimNode {.compileTime.} =
+var registeredEnums {.compileTime.}: HashSet[string]
```

Comment:
This will be a problem in the future. It should be kept in a table with the class name as the key.